### PR TITLE
EVA-509 Hotfix/0.5

### PR DESCRIPTION
**Pull request by pabarcgar**
- Added OSS Jfron artifactory to repositories and distribution managements tags in pom.xml
- Removed biodata build from travis script

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -386,4 +386,16 @@
             </build>
         </profile>
     </profiles>
+    <distributionManagement>
```
Comment (cyenyxe) [inline pom.xml]: Insert blank line before this block.

File: pom.xml [hunk 2]
```diff
@@ -386,4 +386,16 @@
...
+        <snapshotRepository>
+            <id>snapshots</id>
+            <name>oss-jfrog-artifactory-snapshots</name>
```
Comment (cyenyxe) [inline pom.xml]: Agreed offline to use a name based on the project, not on the Artifactory repository.

File: pom.xml [hunk 3]
```diff
@@ -21,21 +21,15 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <biodata.version>0.4.6</biodata.version>
+        <biodata.version>0.4.7</biodata.version>
```
Comment (cyenyxe) [inline pom.xml:24]: The Travis build is complaining about not finding the artifacts in "central" but they seem fine in https://oss.jfrog.org/artifactory/libs-release/uk/ac/ebi/eva/biodata-formats/ ... :thinking:

**Review by cyenyxe — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (pabarcgar) [reply pom.xml:24]: This is fixed

**Review by cyenyxe — APPROVED** [review]

**Pull request merged**